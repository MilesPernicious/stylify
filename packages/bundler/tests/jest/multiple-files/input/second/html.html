<!-- @stylify-pregenerate content:^Excluded^ /@stylify-pregenerate -->

<div class="color:purple"></div>
