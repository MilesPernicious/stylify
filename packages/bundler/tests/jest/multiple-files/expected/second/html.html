<!-- @stylify-pregenerate content:^Excluded^ /@stylify-pregenerate -->

<div class="_5jc7t"></div>
